.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master, v3.0, v3.1, v3.2, v3.3, v3.4, v3.5, v4, v5]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 5 * * 3'
```

Comment:
Does this `cron` entry enables additional testing of official branches that were not modified? If yes, do we really want to run these regular tests for the code already tested on push events above?